src/components/CertificateDropZone/Views/UnverifiedView.test.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
```

Comment:
Using snapshot testing make it not obvious on the test expectations. This article explain some interesting things => https://medium.com/@tomgold_48918/why-i-stopped-using-snapshot-testing-with-jest-3279fe41ffb2

I would suggest we dont use snapshot testing and clearly define our expectations from the test (text renderer for instance)

Reply:
removed snapshot tests